localstack-core/localstack/services/stepfunctions/asl/component/state/state_execution/state_task/service/state_task_service_callback.py

```diff
@@ -334,6 +328,9 @@ def _after_eval_execution(
         normalised_parameters: dict,
         state_credentials: StateCredentials,
     ) -> None:
+        # TODO: In Mock mode, when simulating a failure, the mock response is handled by
```

Comment:
praise: Thanks for clarifying this parity limitation, specifying where and why clearly 👍